### PR TITLE
Fix web-burner cluster-density workload

### PR DESCRIPTION
**Pull request by josecastillolema**
## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Closes https://github.com/kube-burner/kube-burner-ocp/issues/60

TODO: Add test cases for the web-burner-cluster-density scenario, today the CI only covers web-burner-node-density

### CODE REVIEW

File: cmd/config/web-burner-cluster-density/node_density_pod_served.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (vishnuchalla) [inline cmd/config/web-burner-cluster-density/node_density_pod_served.yml:1]: Can this filename go with cluster_density* prefix to keep the naming consistent across the directory.

**Comment by vishnuchalla** [comment]
@josecastillolema Looks like files in this PR got missed previously while adding web-burner cluster density workload. I think adding a CI test will prevent us from running into similar problems in future. So I would kindly request you to add a test in this PR itself, if it is not too much of work. Thanks.

**Comment by josecastillolema** [comment]
@vishnuchalla all comments addressed ptal

**Comment by josecastillolema** [comment]
@vishnuchalla kindly reminder, can we have this merged?
Thanks!

**Pull request merged**